identify_and_start_isolating_side_effects.md

```diff
@@ -6,7 +6,7 @@ General idea
 
 Things outside of your code that affect —or are affected by— the behaviour of your code are called side-effects of your code. _(Calling the former a side-effect may be a bit of a stretch but they both can be treated very similarly to our purposes.)_
 
-Testing code with side-effects is trickier than testing code without side-effects and requires a different approach. Besides, side-effects often mark the boundaries between your code and its dependencies, these are places where change might need to happen without notice.
+Testing code with side-effects is trickier than testing code without side-effects and requires a different approach. Also,side-effects often mark the boundaries between your code and its dependencies. These are places where change might need to happen without notice.
```

Comment:
I am not sure what this line means: These are places where change might need to happen without notice.

Reply:
I mean that changes can be unpredictable, but I was trying not to make it sound like completely erratic. Your dependencies are out of your control, and when they change you need to adapt on your end to follow the change. But, you can predict that they will change someday, it's just that you don't necessarily get a lot of heads up when that finally happens.

Reply:
Ah, that makes sense. So something like: 
"Testing code with side-effects is trickier than testing code without side-effects and requires a different approach. Another thing to keep in mind is that your dependencies are out of your control, and when they change you need to adapt on your end to follow the change. Howeverm you can predict that they will change someday and side-effects often mark the boundaries between your code and its dependencies. The side effects gives us insight where change might need to happen.

Reply:
I'll give it a try! :slightly_smiling_face: